# Yaml file containing the transformations between the the aruco markers and the frames of interest
# The transformations are expressed in the frame of the aruco markers
# define the transform of the frame wrt the aruco marker
resin_block:
  - aruco_id: 0
    frame_transform: 
      translation: [0.000378, 0.00939244, -0.0187691] 
      quaternion: [0.0, -0.981, -0.17719, 0.0] # [qx, qy, qz, qw] 
    name: "side1"
  - aruco_id: 1
    frame_transform:
      translation: [0.0, 0.0, 0.0]
      quaternion: [0.0, 0.0, 0.0, 1.0]
    name: "side2"
  - aruco_id: 2
    frame_transform:
      translation: [0.0, 0.0, 0.0]
      quaternion: [0.0, 0.0, 0.0, 1.0]
    name: "side3"
  - aruco_id: 3
    frame_transform:
      translation: [0.0, 0.0, 0.0]
      quaternion: [0.0, 0.0, 0.0, 1.0]
    name: "side4"


